src/_repobee/cli/mainparser.py

```diff
@@ -13,6 +13,7 @@
 
 from typing import Union, Callable, Optional, Any
 from _repobee.cli import preparser
+from _repobee.command.repos import DirectoryLayout
```

Comment:
It seems a bit weird to import something from the command package into the primary parser, so the `DirectoryLayout` should probably move somewhere else. Not sure where though.

Reply:
Yeah. i would be too inexperienced with repobee core to have any suggestions

Reply:
I've decided to just move everything related to figuring out local repo paths into its own module.